README/MATH/Prediction_Market_Math.tex

```diff
@@ -0,0 +1,1379 @@
+\documentclass{article}
```

Comment:
Thank you for including the TeX file also - this really helps with readability + commentability

Reply:
I'm sorry about this but I'm authoring this document in Overleaf. So it's hard for me to intake the code from Github and copy and paste it back into Overleaf and then maintain it on Github again but I will try. What might be better is to hold off on edits until we have the final math and paper figured out, then go through a final revision. However comments along the way are appreciated and I can update the document to reflect comments, it's just that code changes are a bit harder to keep straight.